inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/util/ExtractNodeUtils.java

```diff
@@ -181,6 +182,9 @@ public static KafkaExtractNode createExtractNode(KafkaSource kafkaSource) {
             case DEBEZIUM_JSON:
                 format = new DebeziumJsonFormat();
                 break;
+            case RAW:
+                format = new RawFormat();
+                break;
```

Comment:
Maybe, you need modify tableOptions of `KafkaExtractNode` if you add this format.

Reply:
Done, it has been removed.